core/src/test/java/org/jdbi/v3/core/extension/TestExtensionContext.java

```diff
@@ -59,8 +60,8 @@ public interface TestExtension {
         Handle getHandle();
 
         default void checkContextInDefaultMethod() throws Exception {
-            Method m = TestExtension.class.getMethod("checkContextInDefaultMethod");
-            // there is not extension method set in default methods. These methods
+            Method m = JdbiClassUtils.methodLookup(TestExtension.class, "checkContextInDefaultMethod");
```

Comment:
This change seems to add another layer of abstraction to this code without really making a specific improvement.

Reply:
mechanic search-and-replace. :-)